tests/data/data_loaders/pytorch_dataloader_test.py

```diff
@@ -20,10 +20,13 @@ def _read(self, filename: str) -> Iterable[Instance]:
             for i in range(NUM_INSTANCES):
                 yield Instance({"index": LabelField(i, skip_indexing=True)})
 
-    reader = FakeDatasetReader(lazy=lazy)
-    dataset = reader.read("blah")
-
-    loader = PyTorchDataLoader(dataset, batch_size=BATCH_SIZE, batches_per_epoch=BATCHES_PER_EPOCH)
+    loader = PyTorchDataLoader.from_partial_objects(
```

Comment:
If you're not actually using any `Lazy` objects, you shouldn't need to call this function.  Why not just call the constructor?